addons/website/static/src/interactions/popup/popup.js

```diff
@@ -22,6 +22,14 @@ export class Popup extends Interaction {
         "_window": {
             "t-on-hashchange": this.onHashChange,
         },
+        ".modal": {
+            // Here, bootstrap's data-bs-backdrop attribute isn't used
+            // and instead we use a custom click handler to dismiss the
+            // popup on click outside as we don't use bootstrap native backdrop.
+            // (see comment above .modal-backdrop in
+            // addons/web/static/src/scss/bootstrap_review.scss).
```

Comment:
Max 80 chars per line for comments... but use them. You break too soon in the first two lines for example.